clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -153,30 +168,49 @@ func (a *apiResponsivenessGatherer) gatherAPICalls(executor QueryExecutor, start
 		if latencyMeasurementDuration < time.Minute {
 			latencyMeasurementDuration = time.Minute
 		}
-		promDuration := measurementutil.ToPrometheusTime(latencyMeasurementDuration)
+		duration := measurementutil.ToPrometheusTime(latencyMeasurementDuration)
 
-		query := fmt.Sprintf(latencyQuery, filters, promDuration)
+		query := fmt.Sprintf(latencyQuery, filters, duration)
 		latencySamples, err = executor.Query(query, measurementEnd)
 		if err != nil {
 			return nil, err
 		}
 	}
 
-	timeBoundedCountQuery := fmt.Sprintf(countQuery, filters, measurementutil.ToPrometheusTime(measurementDuration))
-	countSamples, err := executor.Query(timeBoundedCountQuery, measurementEnd)
+	query := fmt.Sprintf(countQuery, filters, promDuration)
+	countSamples, err := executor.Query(query, measurementEnd)
 	if err != nil {
 		return nil, err
 	}
-	return newFromSamples(latencySamples, countSamples)
+
+	allowedSlowCalls, err := util.GetIntOrDefault(config.Params, "allowedSlowCalls", 0)
+	if err != nil {
+		return nil, err
+	}
+
+	countSlowSamples := make([]*model.Sample, 0)
+	if allowedSlowCalls != 0 {
```

Comment:
Actually, I think it would be useful to propagate that no matter if this is going to be later used or not - this metric is pretty interesting by itself.